src/components/wallet/earn/DateForm.vue

```diff
@@ -144,7 +144,7 @@ export default class DateForm extends Vue {
         let start = this.localStart
         let startDate = new Date(start)
 
-        let end = startDate.getTime() + DAY_MS * 21
+        let end = startDate.getTime() + DAY_MS * 1
```

Comment:
Why this is changed ?

Reply:
because on previously on date placeholder it was showing the date  21 days to current date